src/App.tsx

```diff
@@ -1,61 +1,129 @@
+import { ChangeEvent, useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { ITodoInfo } from './types/ITodoInfo';
+
+const initialTodos: ITodoInfo[] = todosFromServer.map(todo => {
+  const user = usersFromServer.find(us => us.id === todo.userId) as User;
+
+  return { ...todo, user };
+});
 
 export const App = () => {
+  const [todos, setTodos] = useState<ITodoInfo[]>(initialTodos);
+  const [hasTitleError, setHasTitleError] = useState(false);
+  const [hasSelectedIdError, setHasSelectedIdError] = useState(false);
+
+  const [selectedUserId, setSelectedUserId] = useState<number | null>(null);
+  const [title, setTitle] = useState('');
+
+  const getTodoId = (todoArr: ITodoInfo[]) => {
+    const idMax = Math.max(...todoArr.map((todo: ITodoInfo) => todo.id));
+
+    return idMax + 1;
+  };
+
+  const handlerSubmit = (e: React.FormEvent) => {
```

Comment:
```suggestion
  const handleSubmit = (e: React.FormEvent) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364